ios/browser/api/net/certificate_utility.mm

```diff
@@ -0,0 +1,143 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#import "brave/ios/browser/api/net/certificate_utility.h"
+
+#include <algorithm>
+#include <memory>
+#include <tuple>
+#include <utility>
+#include <vector>
+
+#include "base/base64.h"
+#include "base/bind.h"
+#include "base/logging.h"
+#include "base/memory/ref_counted.h"
+#include "base/strings/sys_string_conversions.h"
+
+#include "net/cert/internal/cert_errors.h"
+#include "net/cert/internal/parsed_certificate.h"
+#include "net/cert/pem.h"
+#include "net/cert/x509_certificate.h"
+#include "net/cert/x509_util.h"
+#include "net/der/input.h"
+#include "net/http/hsts_info.h"
+#include "net/http/transport_security_state.h"
+#include "net/net_buildflags.h"
+#include "net/tools/transport_security_state_generator/cert_util.h"
+#include "net/tools/transport_security_state_generator/spki_hash.h"
+
+#include "third_party/boringssl/src/include/openssl/mem.h"
+#include "third_party/boringssl/src/include/openssl/sha.h"
+#include "third_party/boringssl/src/include/openssl/x509.h"
+
+namespace net {
+namespace {
+#if BUILDFLAG(INCLUDE_TRANSPORT_SECURITY_STATE_PRELOAD_LIST)
+#include "net/http/transport_security_state_static.h"  // nogncheck
```

Comment:
Is there a reason we skip gn check on this include?

Reply:
This is a straight copy from Chromium: https://source.chromium.org/chromium/chromium/src/+/main:net/http/transport_security_state.cc;l=55?q=INCLUDE_TRANSPORT_SECURITY_STATE_PRELOAD_LIST&sq=&ss=chromium%2Fchromium%2Fsrc